src/views/messages/message-rows/follow-user.jsx

```diff
@@ -29,7 +29,6 @@ var FollowUserMessage = React.createClass({
                     values={{
                         profileLink: <a
                             href={profileLink}
-                            className="social-messages-profile-link"
```

Comment:
I wonder if removing this class is the best solution? After all, this *is* still a profile link. Maybe having a separate class - something like `.social-messages-focus-link` (on all links that are meant to be "focused", such as who followed you, or a link to a project or studio) - would be better.

Reply:
I was wondering the same thing, but (as stated in the OP) I noticed that it was *specifically that class* that made it look like normal text instead of a link - suggesting that it was originally meant to be that way. I could just change the style of that class - but I can't seem to find the root style for links on the messages page. I guess I'll work on that then.